### DISCUSSION TITLE
ECMA-419 standard

### DISCUSSION THREAD

**Discussion by bartmichu** [microsoft/devicescript#488] [General]
Hello.

Did you came across the [ECMA-419 standard](https://419.ecma-international.org/)? It is for embedded systems API specification - I think it would make a lot of sense to adopt [some of] that in DeviceScript. What is your take on this idea? 

Your project looks really promising so keep up the good work and thank you!

**Comment by bartmichu** [comment]
Also, here is a link for [2nd edition draft and more resources](https://github.com/EcmaTC53/spec)

**Comment by mmoskal** [comment]
Currently we have no plans to implement this. Maybe it could be implemented as a library externally if there's interest?
